test/runtests.jl

```diff
@@ -305,4 +305,37 @@ facts("High order polynomials test inspired by Fateman (takes a few seconds))")
     @fact get_coeff(f2,[1,6,7,20]) == c --> true
 end
 
+facts("Matrix multiplication for Taylor1") do
+    order = 30
+    n1 = 100
+    k1 = 90
+
+    tol1 = eps(1.0)
+
+    fake_order = max(n1,k1,order+1)
```

Comment:
`fake_order` should be identical to `order+1`; it is related to the order of the `Taylor1` polynomials. Or did I miss something?